pandasai/__init__.py

```diff
@@ -323,12 +325,15 @@ def _is_unsafe_import(self, node: ast.stmt) -> bool:
 
         if isinstance(node, (ast.Import, ast.ImportFrom)):
             for alias in node.names:
-                if alias.name in WHITELISTED_BUILTINS:
+                if (
+                    alias.name in WHITELISTED_BUILTINS
+                    or alias.name in ENVIRONMENT_DEFAULTS
```

Comment:
I'm not 100% satisfied with the implementation. But it works as a hotfix.

Reply:
I guess it's ok for now. We'll need to refactor it all at a later point!